src/main/java/com/raisetech10/liquor_management/mapper/LiquorMapper.java

```diff
@@ -26,9 +23,13 @@ public interface LiquorMapper {
     //PATCH
     @Select("Select * FROM liquor WHERE id = #{id}")
     Optional<Liquor> findLiquorId(int id);
-    
+
     @Update("UPDATE liquor SET liquorType = #{liquorType}, producingCountry = #{producingCountry}, liquorName = #{liquorName}, alcoholContent = #{alcoholContent} WHERE id = #{id}")
     void update(Liquor liquor);
 
+    //DELETE
+    @Delete("DELETE FROM liquor WHERE id = #{id}")
+    void delete(int id); //void deleteLiquor(int id);
```

Comment:
`//void deleteLiquor(int id);`要りますか？
```suggestion
    void delete(int id);
```

Reply:
不用でした。削除します。